### PR TITLE
Document on how to use Argo CD for managing OpenShift cluster configurations across multiple clusters

### PR DESCRIPTION
**Pull request by Srivaralakshmi**
**Aligned team**: Dev Tools

**Purpose**: To resolve the following issues:
https://issues.redhat.com/browse/RHDEVDOCS-3604

**OCP version this PR applies to**: `4.8` and later

**Link to docs preview**: 
- [Using an Argo CD instance to manage cluster-scoped resources](https://59008--docspreview.netlify.app/openshift-enterprise/latest/cicd/gitops/configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.html#using-argo-cd-instance-to-manage-cluster-scoped-resourcesconfiguring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations)
- [Default permissions of an Argocd instance](https://59008--docspreview.netlify.app/openshift-enterprise/latest/cicd/gitops/configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.html#default-permissions-of-an-argocd-instance.adocconfiguring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations)
- [Installing OLM Operators using Red Hat OpenShift GitOps](https://59008--docspreview.netlify.app/openshift-enterprise/latest/cicd/gitops/configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.html#gitops-installing-olm-operators-using-gitops_configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations)

**SME review**: Completed by @iam-veeramalla  
**QE review**: Completed by @varshab1210
**Peer-review**:  Completed by @deerskindoll

### CODE REVIEW

File: modules/gitops-installing-olm-operators-using-gitops.adoc [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  installPlanApproval: Automatic
+  name: ansible-automation-platform-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+...
+----
+
+[IMPORTANT]
+====
+When deploying multiple Operators using {gitops-title}, you must create only a single Operator group in the corresponding namespace. If more than one Operator group exists in a single namespace, any CSV created in that namespace transition to a `failure` state with the `TooManyOperatorGroups` reason. After the number of Operator groups in their corresponding namespaces reaches one, all the previous `failure` state CSVs transition to `pending` state. You must manually approve the pending install plan to complete the Operator installation.
```
Comment (Srivaralakshmi) [inline modules/gitops-installing-olm-operators-using-gitops.adoc:78]: > You must manually approve the pending install plan to complete the Operator installation.

@iam-veeramalla, @varshab1210 Is it ok to include this statement?

File: modules/gitops-using-argo-cd-instance-to-manage-cluster-scoped-resources.adoc [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+[TIP]
+====
+Alternatively, in the OpenShift CLI, run the following command:
+
+[source,terminal]
+----
+oc auth can-i create oauth -n openshift-gitops --as system:serviceaccount:openshift-gitops:openshift-gitops-argocd-application-controller
+----
+
+The output `yes` verifies that the Argo instance is configured with a cluster role to manage cluster-scoped resources. Else, check your configurations and take necessary steps as required.
```
Comment (Srivaralakshmi) [inline modules/gitops-using-argo-cd-instance-to-manage-cluster-scoped-resources.adoc:50]: > Else, check your configurations and take necessary steps as required.

@iam-veeramalla, @varshab1210 Is it ok to include this statement?

**Comment by Srivaralakshmi** [comment]
@iam-veeramalla and @varshab1210: PTAl at the PR and provide feedback. Thanks!

File: modules/gitops-installing-olm-operators-using-gitops.adoc (hunk 1 above)
Comment (Srivaralakshmi) [reply modules/gitops-installing-olm-operators-using-gitops.adoc:78]: @iam-veeramalla, @varshab1210: Can we include this link in the Additional resources section:

https://docs.openshift.com/container-platform/4.11/operators/admin/olm-adding-operators-to-cluster.html

**Comment by Srivaralakshmi** [comment]
@iam-veeramalla: To which OCP version do we CP this PR content? The [GITOPS-370](https://issues.redhat.com/browse/GITOPS-370) issue shows the Fix version as `1.5` whose supported OCP version is `4.8`-`4.11`.

- `4.8`- `4.11`?
- Or `4.10` and later?
- Or `4.8` and later?

**Comment by varshab1210** [comment]
IMO, its okay to include the statements you have mentioned. 

/lgtm

**Review by deerskindoll — APPROVED** [review]
lgtm

**Pull request merged**